### DISCUSSION TITLE
VMXPackager Error:

### DISCUSSION THREAD

**Discussion by rongzhonghuang** [amazon-connect/amazon-connect-salesforce-scv#135] [Q&A]
Hi, I followed the instruction to implement Salesforce Custom Objects, but I got error in the VMXPackager logs as below, could someone know how to resolve the issue? 

[ERROR]	2022-09-22T12:48:09.304Z	b33d9db1-1658-4dc8-95c5-bab30c4cc786	Record 1 Result: Failed to generate presigned URL

**Comment by dougjaso** [comment]
That's not enough, really, to go on. Can you give me the logs from the presigner function?
